src/types/relocatable.rs

```diff
@@ -256,7 +256,9 @@ impl MaybeRelocatable {
                 MaybeRelocatable::RelocatableValue(rel_b),
             ) => {
                 if rel_a.segment_index == rel_b.segment_index {
-                    return Ok(MaybeRelocatable::from(Felt252::new((*rel_a - *rel_b)?)));
+                    return Ok(MaybeRelocatable::from(Felt252::from(
+                        rel_a.offset as i128 - rel_b.offset as i128,
```

Comment:
I think it should be better to leave this implementation as it is and move this modification to the Relocatable sub Relocatable operation. So we don't have two different implementations of the Relocatable sub.
Do you think it is ok?

Reply:
I think I was wrong on this one
the Relocatable - Relocatable returns an usize